packages/test/test-end-to-end-tests/src/test/cellEndToEndTests.spec.ts

```diff
@@ -279,6 +280,43 @@ describeNoCompat("SharedCell orderSequentially", (getTestObjectProvider) => {
         containerRuntime = dataObject.context.containerRuntime as ContainerRuntime;
     });
 
+    it("Should rollback set", async () => {
```

Comment:
Add another case to rollback to a previous value. And to rollback multiple edits to the same cell in the callback.